sbndprmdaq/manager.py

```diff
@@ -556,6 +562,41 @@ def save_data(self, prm_id=1):
 
         self._logger.info(f'Data saved for PrM {prm_id}.')
 
+        return out_dict
+
+    def output_to_epics(self, prm_id, out_dict):
+        '''
+        Updates EPICS with run data.
+
+        Args:
+            prm_id (int): The purity monitor ID.
+            out_dict (dict): Data from run.
+        '''
+        if prm_id == 1:
+            prm = 'tpcshort'
+        elif prm_id == 2:
+            prm = 'tpclong'
```

Comment:
Please invert long and short. I also inverted the labels on the gui in PR#25. Sorry I had them flipped to begin with

Reply:
Ok, I swapped them